samples/dockercompose/ingress.yml

```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1
```

Comment:
FYI this is a Kubernetes 1.19 feature. The default on AKS is 1.18.10 for westus2 at the time of checking. I found this notable because there's a few features in 1.19 that Tye *could* take advantage of for ingress, but we didn't do it when I wrote this because the paint wasn't quite dry yet.

Reply:
Specifically in 1.19 `IngressClass` is a resource type on its own: https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-class

This means that you can query the cluster for existing `IngressClass` definitions, and use them without being coupled to nginx-ingress.  The whole set of features related to detecting the presence of nginx-ingress would be replaced by a lookup of a documented resource type 😆 - and Tye can start leveraging other ingress controllers.

Reply:
I thought it was 1.19... I'll try verifying this with 1.18 to see what happens.

Reply:
And *very cool*